src/openms/source/CHEMISTRY/EnzymaticDigestion.cpp

```diff
@@ -171,20 +160,56 @@ namespace OpenMS
     log_model_threshold_ = threshold;
   }
 
+  std::vector<Size> EnzymaticDigestion::tokenise(const AASequence& protein) const
```

Comment:
might be better to use const std::string& here instead of the relativly slow AASequence object. This should be possible (see comment below)